pages/recommend.js

```diff
@@ -0,0 +1,93 @@
+import Head from "next/head";
+import styles from "../styles/Recommend.module.css";
+import React from "react";
+import SearchBar from "../components/home/SearchBar";
+import BookCard from "../components/recommended/BookCard";
+import { useRouter, withRouter } from "next/router";
+
+export default withRouter(
```

Comment:
Using withRouter as wrapper... Got it from https://stackoverflow.com/a/67637875, dont trust to much